packages/validators/typescript/src/TypeScriptValidator.js

```diff
@@ -45,21 +48,28 @@ export default new Validator({
     if (tsconfig && !langServiceCache[configHash]) {
       let parsedCommandLine = ts.parseJsonConfigFileContent(
         tsconfig,
-        ts.sys,
+        new ParseConfigHost(options.inputFS, ts),
```

Comment:
Note: this is a new change that I had to make in order to get unit tests working. Without it, typescript will not look to the `inputFS` specified in parcel configuration (which is injected in unit tests so we can modify it programmatically).

Reply:
Awesome, it should have been like that all along, thanks for fixing this bug